packages/terra-button-group/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+
+*Fixed
+  *Fixed visual focus When using SR and added Cyclic Behavior for terra button group.
+
```

Comment:
Corrections:
- Since you're adding a new feature, that should be a separate entry under `Added`.
- Screenreader should be fully spelled out
- Markdown syntax requires a space after `*` for bullet points.
- "Cyclic Behavior" is not a proper noun and does not need to be capitalized.
- This is a dedicated CHANGELOG file for terra button group so we don't need to explicitly state that.

```suggestion

* Fixed
  * Fixed visual focus when using screenreaders.
  
 * Added
   * Added cyclic behavior when navigating through buttons with a keyboard.

```

Reply:
Updated doc file - [1b80e6a](https://github.com/cerner/terra-core/pull/3857/commits/1b80e6a33f8584d5fb55dfb752f2bf6c005407f5)